src/platforms/python/gcp_functions.mdx

```diff
@@ -17,24 +18,50 @@ You can use the GCP Functions integration for the Python SDK like this:
 
 ```python
 import sentry_sdk
-from sentry_sdk.integrations.serverless import serverless_function
+from sentry_sdk.integrations.gcp import GcpIntegration
 
-sentry_sdk.init(dsn="___PUBLIC_DSN___")
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    integrations=GcpIntegration()]
+)
 
-@serverless_function
-def my_function(...): ...
+def my_function(event, context):
+    ...
 ```
 
-Use the generic integration by calling the `serverless_function` decorator. Decorators wrap a function and modify its behavior. Then, deploy and test the function. Checkout Sentry's [sample apps](https://github.com/getsentry/examples) for detailed examples.
+Checkout Sentry's [GCP sample apps](https://github.com/getsentry/examples/tree/master/gcp-cloud-functions) for detailed examples.
+
+### Enable Timeout Warning
```

Comment:
We should explain what the timeout warning does before instructing users to enable it